ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjectsPhysical.java

```diff
@@ -23,7 +23,7 @@
  * Use the DetailedObjectsPhysicalView to format output
  * by physical placement: one line for each instance on tape
  */
-public class GetFilteredObjectsPhysical extends GetFilteredObjects {
+public class GetDetailedObjectsPhysical extends GetDetailedObjects {
```

Comment:
Command name change should be updated in resources: com.spectralogic.ds3cli.command.CliCommand and com.spectralogic.ds3cli.help.properties